inc/Engine/Optimization/Minify/JS/AbstractJSOptimization.php

```diff
@@ -101,7 +101,13 @@ protected function is_minify_excluded_file( array $tag ) {
 		}
 
 		// File should not be minified.
-		if ( false !== strpos( $tag[0], 'data-minify=' ) || false !== strpos( $tag[0], 'data-no-minify=' ) ) {
+		if (
+			false !== strpos( $tag[0], 'data-minify=' )
+			||
+			false !== strpos( $tag[0], 'data-no-minify=' )
+			||
+			false !== strpos( $tag[0], 'data-rocketlazyloadscript=' )
```

Comment:
This check should be in the combine JS class specifically, because having the file minified is fine (it stays in place and can be delayed the same), but having it combined (and thus, removed from the HTML) is not